pkg/api/component_report.go

```diff
@@ -1382,12 +1393,12 @@ func (c *componentReportGenerator) generateComponentTestReport(baseStatus map[ap
 		delete(sampleStatus, testIdentification)
 
 		rowIdentifications, columnIdentifications := c.getRowColumnIdentifications(testIdentification, baseStats)
-		updateCellStatus(rowIdentifications, columnIdentifications, testID, reportStatus, aggregatedStatus, allRows, allColumns)
+		updateCellStatus(rowIdentifications, columnIdentifications, testID, reportStatus, sampleStats, aggregatedStatus, allRows, allColumns)
```

Comment:
The report status gets modified by adjusting the sample stats in assessComponentStatus.  Those adjustments won't be reflected here but I think that will be ok if we are ok with my comment @ line 1018

Reply:
Good catch. We might need to add the compensation amount to the sampleStats struct to have proper numbers reflected in the regressedTests. 

While looking at the code, I wonder if there is a potential miscount involving the compensation calculation. It seems to me that the triaged compensation amount is based on jobRuns involved. But the sample stats is test based. That means, if a test appears multiple times in a job run, it is only counted once for compensation. This should be common in an upgrade job.

Reply:
My other comment was that we won't be storing the sampleStatus unless the reportStatus is still considered a regression.  But granted if we made an adjustment and were still considered regressed that adjustment would not be reflected in the sampleStats.  Still think we are ok for our current needs and keeping the 'real' values may in fact be desirable.

Curious about your second comment.  We might need to walk through that more though I'm thinking potential exposure is low.  You are saying that the same testID could fail twice in a single job run and only be counted once for triage.  Might be something to review with Stephen, I know he has some deduping logic and it might cover this case so a test failure is only counted once per job run.  Not sure though.

Reply:
If a triaged issue is associated with a job that have multiple potential regressions, do we count the compensation for all of them even if only one is being worked on? If so, the other regressions should still show up in the regressedTests and maybe we do like the current handling.

Let's sync in the office hour in case I am throwing out nonsense here. :)

Reply:
Triaged Regressions are TestID specific.  So if there were 3 TestIDs that were being triaged there would be 3 entries (each with the list of jobruns that were considered triaged for that TestID).

What I don't know is if the same TestID would be reused for conformance & parallel.  I wouldn't think so but can't state definitively without more digging.

Sorry Devan, we hijacked your PR...